test/kitchensink/integration/dev-server.test.js

```diff
@@ -10,4 +10,14 @@ describe('dev-server', () => {
       expect.arrayContaining(['components/', 'app.bundle.js', 'assets/']),
     );
   });
+
+  it('shows the contents of assets dir', async () => {
+    await page.goto('http://localhost:3200/assets');
+
+    const list = await page.$$eval('#files li a', a => {
+      return a.map(li => li.textContent);
+    });
+
+    expect(list).toEqual(expect.arrayContaining(['hello.txt']));
```

Comment:
this can be written also as:

```suggestion
    expect(list).toContain('hello.txt');
```

which is less code and more reasonable IMO. what do you think?

Reply:
```node
> let expect = require('expect')
undefined
> expect([1,2,3]).toContain(1);
undefined
> expect([1,2,3]).toContain(10);
Error: expect(array).toContain(value)

Expected array:
  [1, 2, 3]
To contain value:
  10
```